.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    name: "CI | python v${{ matrix.python-version }}, postgres v${{ matrix.postgres-version }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+        postgres-version: ["12", "13", "14", "15"]
+        django-version: ["3.2", "4.0", "4.1", "4.2"]
```

Comment:
Sorry just one more bit of feedback on the testing matrix, you'll probably want to add a couple "excludes" here since for instance [Django 3.2 doesn't run on Python 3.11](https://docs.djangoproject.com/en/4.2/faq/install/#what-python-version-can-i-use-with-django) 

![2023-04-27_10-22](https://user-images.githubusercontent.com/1423728/234940869-be2ff493-3916-4fb2-9701-c0ca91bfac0b.png)

```suggestion
        django-version: ["3.2", "4.0", "4.1", "4.2"]
        exclude:
          - python-version: "3.11"
            django-version: "3.2"
          - python-version: "3.11"
            django-version: "4.0"
```

You can see an [example of excludes here](https://github.com/kevinmarsh/django-classy-tags/blob/021706f911920e84b43d646b0a50837ea3add156/.github/workflows/test.yml#L11-L21)

Reply:
👍🏽